script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -716,7 +716,7 @@ Someone who, in spite of being human, is born with some form of Magic Circuit: a
 [sha:465b3cfc42588a0003968a493220d397df6b426f]{
 -- Page 479, Offset 17762.
 -- ―――例えば何の神秘も学んでいないのに“物の死”を視てしまう人間とか、ね」
-―――For example, a human who has never studied the Mysteries but is able to see the 'death of things'."
+―――For example, a human who has never studied Mystery but is able to see the 'death of things'."
```

Comment:
not sure but wiki says Mystery (localized as Mystics)

Reply:
Natural inclination is to pluralize a group like this (you can either study a specific Mystery, or [the] Mysteries in general) but @yk1512 I think did a pass on these before